src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -534,6 +534,12 @@
   "DownloadClientStatusSingleClientHealthCheckMessage": "Download clients unavailable due to failures: {downloadClientNames}",
   "DownloadClientTransmissionSettingsDirectoryHelpText": "Optional location to put downloads in, leave blank to use the default Transmission location",
   "DownloadClientTransmissionSettingsUrlBaseHelpText": "Adds a prefix to the {clientName} rpc url, eg {url}, defaults to '{defaultUrl}'",
+  "DownloadClientTriblerSettingsAnonymityLevel": "Anonymity level",
+  "DownloadClientTriblerSettingsAnonymityLevelHelpText": "Number of proxies to use when downloading content. To disable set to 0. Proxies reduce download/upload speed. See {url}",
+  "DownloadClientTriblerSettingsApiKeyHelpText": "[api].key from triblerd.conf",
+  "DownloadClientTriblerSettingsDirectoryHelpText": "Optional location to put downloads in, leave blank to use the default Tribler location",
+  "DownloadClientTriblerSettingsSafeSeeding": "Safe Seeding",
+  "DownloadClientTriblerSettingsSafeSeedingHelpText": "Only Seed through proxies.",
```

Comment:
It will only seed to tribler instances running in the ipv8 network or through ipv8 exit nodes. Much like Tor.
A better description might be:
Anonymously seed through Tribler network: Seed only to Tribler nodes and through anonymous Tribler exit nodes. (Link to https://www.tribler.org/anonymity.html)